privaterelay/templates/newlanding/a/hero.html

```diff
@@ -12,12 +12,26 @@ <h1 class="c-landing-hero--title mzp-c-section-heading">{% ftlmsg 'landing-hero-
         <img alt="" src="{% static 'images/newlanding/a/hero-image-bg.svg' %}" class="c-hero-image--bg"/>
         {% if premium_available_in_country %}
             <img alt="" src="{% static 'images/newlanding/a/hero-image-premium.svg' %}" class="c-hero-image"/>
+
+            {% if accept_language|slice:"0:2" == "fr" %}
+                <!-- French -->
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-premium-fr.svg' %}" class="c-hero-image"/>
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-fg-fr.svg' %}" class="c-hero-image--fg"/>
+            {% elif accept_language|slice:"0:2" == "de" %}
+                <!-- German -->
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-premium-de.svg' %}" class="c-hero-image"/>
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-fg-de.svg' %}" class="c-hero-image--fg"/>
+            {% else %}
+                <!-- Default to English -->
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-premium.svg' %}" class="c-hero-image"/>
+                <img alt="" src="{% static 'images/newlanding/a/hero-image-fg.svg' %}" class="c-hero-image--fg"/>
+            {% endif %}
+
         {% else %}
+            <!-- Non-premium / English only -->
             <img alt="" src="{% static 'images/newlanding/a/hero-image-nopremium.svg' %}" class="c-hero-image"/>
```

Comment:
Edge case: If there's a French/German speaking country where we do not offer premium, they will get a English version of the non-premium image on the homepage.

Reply:
That seems fine to me; the main reason we're doing this is because we're going to be actively reaching out to people in Germany and France, where we do offer Premium. For the rest, this isn't a regression.